pf_driver/include/pf_driver/pf/pfsdp_protocol.hpp

```diff
@@ -362,9 +367,13 @@ class PFSDPBase
     return handle_info;
   }
 
-  virtual HandleInfo request_handle_udp(const std::string host_ip, const std::string port)
+  virtual HandleInfo request_handle_udp(const std::string host_ip, const std::string port, const std::string packet_type = "")
   {
-    auto resp = get_request("request_handle_udp", { "handle", "port" }, { KV("address", host_ip), KV("port", port) });
+    param_map_type query = { KV("address", host_ip), KV("port", port) };
```

Comment:
Might make sense to add some tests later to check for validity of requests URI.

Reply:
I assume that will be a different PR...

Reply:
Yes